main.ts

```diff
@@ -6,6 +6,8 @@ import capitalizeFirstLetter from "Utils/capitalizeFirstLetter";
 export enum SearchProvider {
 	SWITCHER = "switcher:open",
 	OMNISEARCH = "omnisearch:show-modal",
+	QUICKSWITCHER_PLUS = "darlal-switcher-plus:switcher-plus:open", //open in standard mode
+	ANOTHER_QUICKSWITCHER = "obsidian-another-quick-switcher:search-command_file-name-search"
```

Comment:
I don't use Another Quick Switcher but taking a look at their code real quick it seems like the command names are dynamic. 

It it possible that this won't work for someone who has customized the plugin?

Reply:
Yeah, that's the problem with Another Quick Switcher :<
Maybe we need to remove it or add a new option where user select the commands directly in a modal ?

Reply:
Hm. That might be a bit much for this PR. Let's maybe keep this one out for now and open a PR with brainstorming possible options.